Src/ILGPU/Runtime/ExchangeBuffer.cs

```diff
@@ -177,17 +179,33 @@ protected override void Dispose(bool disposing)
         /// </summary>
         public TIndex Extent { get; protected set; }
 
+        /// <summary>
+        /// Internal array view
+        /// </summary>
+        [SuppressMessage(
+            "Microsoft.Design",
+            "CA1051: Do not declare visible instance fields",
+            Target = "target")]
+        protected ArrayView<T, TIndex> cpuView;
+
         /// <summary>
         /// Returns an array view to the CPU part of this buffer.
         /// </summary>
-        public ArrayView<T, TIndex> CPUView { get; protected set; }
+        [Obsolete("Replaced by Span property")]
+        [CLSCompliant(false)]
+        public ArrayView<T, TIndex> CPUView => cpuView;
+
+        /// <summary>
+        /// Returns a span to the part of this buffer in CPU memory
```

Comment:
MINOR: Missing full stop.

```suggestion
        /// Returns a span to the part of this buffer in CPU memory.
```